test/unit/nedap/utils/test/matchers.clj

```diff
@@ -0,0 +1,20 @@
+(ns unit.nedap.utils.test.matchers
+  (:require
+   [clojure.test :refer [deftest testing are is]]
+   [matcher-combinators.standalone :refer [match]]
+   [nedap.utils.test.matchers :as sut])
+  (:import
+   (clojure.lang ExceptionInfo)))
+
+(deftest in-any-order
+  (testing "behaves like #'matcher-combinators.matchers/in-any-order"
+    (are [actual expected] (= expected (match (sut/in-any-order (range 8)) actual))
+      (shuffle (range 8))
+      {:match/result :match}
+
+      (range 7)
+      {:match/result :mismatch, :mismatch/detail '(6 5 4 3 2 1 0 #matcher_combinators.model.Missing{:expected 7})}))
+
+  (testing "will time out after some time"
```

Comment:
you could also author a test that actually is time-based (e.g. `Thread/sleep`), `are`ing 40 vs. 50

Reply:
No real clear way to do so. had some elaborate setup +  obv there's overhead to account for which made the test look strange.

Nothing at all isn't a great alternative though 😅

Reply:
Thought about this; the contrast is already here.

The top-are case showcases an example which does not timeout. The second `testing`-block does time-out because `in-any-order` is eagerly trying to find a unsatisfiable match (which is exactly the behaviour we want to circumvent with this wrapper).